### DISCUSSION TITLE
How to Sort results by attribute?

### DISCUSSION THREAD

**Discussion by monkemedia** [nuxt-modules/algolia#157] [Q&A]
Can anyone help me? I would like to sort my results by attribute. For example, I would like to sort my results by price (desc or asc). How would I go about this?

sadly this doesn't work:

```
await search({
      query: route.params.searchQuery,
      requestOptions: {
        page: page.value - 1,
        hitsPerPage: 40,
        ranking: ['price(desc)']
      }
    })

This is the error I get:

Invalid value for "ranking" parameter, expected Invalid ranking parameter: "pricedesc", you should use
typo,geo,words,filters,proximity,attribute,exact,custom or asc(attribute),desc(attribute)

**Comment by Baroshem** [comment]
Hey, 

have you tried the opposite ranking values: `desc(price)`?

It seems like price or pricedesc is not an available ranking option

**Reply by monkemedia** [reply]
sadly this doesn't work either

**Reply by Baroshem** [reply]
Are you sure that this filter option is available in the version of algoliasearch that is used in the module?

Maybe it was added later and it is not available yet.

Other than that, I dont have actually any other ideas :(
